test/interceptors/dns.js

```diff
@@ -546,31 +550,13 @@ test('Should we handle TTL (4)', async t => {
     dns({
       dualStack: false,
       affinity: 4,
-      maxTTL: 100,
+      maxTTL: 400,
       lookup: (origin, opts, cb) => {
         ++lookupCounter
         lookup(
           origin.hostname,
           { all: true, family: opts.affinity },
-          (err, addresses) => {
-            if (err) {
-              return cb(err)
-            }
-
-            const results = new Map()
-
-            for (const addr of addresses) {
-              const record = {
-                address: addr.address,
-                ttl: opts.maxTTL,
-                family: addr.family
-              }
-
-              results.set(`${record.address}:${record.family}`, record)
-            }
-
-            cb(null, results.values())
-          }
```

Comment:
why was this deleted?

Reply:
I didn't think it adds any value to the test. Does it?